src/oscar/apps/voucher/abstract_models.py

```diff
@@ -170,10 +170,8 @@ def __str__(self):
         return self.name
 
     def clean(self):
-        if all([self.start_datetime, self.end_datetime,
-                self.start_datetime > self.end_datetime]):
-            raise exceptions.ValidationError(
-                _('End date should be later than start date'))
+        if self.start_datetime and self.end_datetime and self.start_datetime > self.end_datetime:
```

Comment:
This is also getting added in [#3632](https://github.com/django-oscar/django-oscar/pull/3632/files#diff-996e1c61247748a36e9dbf3c9847d19ed403a2459a8bfcbab1bb35f3fa180044L173-R143).

Reply:
Oh I'd missed that - let's just wait for that.